cmake/SetupDocs.cmake

```diff
@@ -77,29 +77,30 @@ macro(blt_add_sphinx_target sphinx_target_name )
     # HTML output directory
     set(SPHINX_HTML_DIR "${CMAKE_CURRENT_BINARY_DIR}/html")
 
-    # support both direct use of a conf.py file and a cmake config-d 
-    # sphinx input file (conf.py.in)
-    if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/conf.py")
+    # support both direct use of a conf.py file and a cmake config-d
```

Comment:
> config-d

Is this an abbreviation for ``configured``? Or does it mean something else?

If it is the former, I think it would be clearer to change it to ``configured``.

In  this context, we are discussing files with names ``config.py`` and ``config.py.in`` and I struggled a bit to parse the comment.

Reply:
This was an existing comment 😄 I agree that "configured" is better.